go-controller/pkg/libovsdb/ops/chassis.go

```diff
@@ -151,10 +151,10 @@ func CreateOrUpdateChassis(sbClient libovsdbclient.Client, chassis *sbdb.Chassis
 			BulkOp:         false,
 		},
 		{
-			Model:          chassis,
-			OnModelUpdates: onModelUpdatesAllNonDefault(),
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model:            chassis,
+			OnModelMutations: []interface{}{&chassis.OtherConfig},
```

Comment:
Any reason not to mutate the `chassis.Encaps` as well? If so, I would add a comment for that.